hw2/05-form.js

```diff
@@ -1,15 +1,43 @@
 // Add your code here
-const userName = document.getElementById('InputName').value
 const submit = document.getElementById('Submit');
 const reset = document.getElementById('Reset');
 
+submit.addEventListener('click', ()=>{
 
-console.log(userName)
+    const userName = document.getElementById('InputName').value
+    const email = document.getElementById('InputEmail').value
+    const registration = document.getElementById('Registration').value
+    const progCheck = document.getElementById('ProgrammingLang').checked
+    const opCheck = document.getElementById('OperatingSystems').checked
+    const fullCheck = document.getElementById('FullStack').checked
+    const anything = document.getElementById('Anything').value
+    console.log(userName)
+    console.log(email)
+    console.log(registration)
 
-submit.addEventListener('click', ()=>{
+    if(progCheck === true){
+        console.log("Programming Languages")
+    }
+    if(opCheck === true){
+        console.log("Operating Systems")
+    }
+    if(fullCheck === true){
+        console.log("Full Stack")
+    }
+
+    if(anything){
+        console.log(anything)
+    }
+    
 
 })
 
 reset.addEventListener('click', ()=>{
-    
+    document.getElementById("InputName").value = ""
+    document.getElementById('InputEmail').value = ""
+    document.getElementById('Registration').value = "Choose an Option"
+    document.getElementById('ProgrammingLang').checked = false
+    document.getElementById('OperatingSystems').checked = false
+    document.getElementById('FullStack').checked = false
```

Comment:
You might want to use document.getElementByName property for courses. That way we don’t have to get all the options by ID.